create_cwl_from_objects.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+import cwl_utils.parser_v1_2 as cwl
+
+from ruamel import yaml
+
+def main() -> None:
+    """Generate a CWL object to match "cat-tool.cwl"."""
+    inputs = [cwl.WorkflowInputParameter(id="input_file", type="File",)]
+    outputs = [
+        cwl.WorkflowOutputParameter(
+            id="output_file",
+            type="File",
+            outputSource="md5sum/output_file",
+            secondaryFiles='md5sum.cwl'
+        )
+    ]
+    steps = [
+        cwl.WorkflowStep(
+            id = "md5sum",
+            run = "dockstore-tool-md5sum.cwl",
+            in_ = "input_file: input_file",
```

Comment:
Hmm.. I think you need to use a `WorkflowStepInput` here https://github.com/common-workflow-language/cwl-utils/blob/53d415cfaf2082a24bad7e5c1ceaabe8ea9a799e/cwl_utils/parser_v1_2.py#L10331

Reply:
`WorkflowStepInput` is not taking the 'out' parameter. isn't it?

Reply:
I'm getting 'Tool definition is valid'  after running `cwltool --validate result.cwl`, but result.cwl is a bit different from  https://github.com/common-workflow-language/cwl-utils/blob/main/testdata/md5sum.cwl. What shall I do next? Shall I make a PR with the new python file and the .cwl file?

Reply:
Yes, please make a PR

Reply:
Actually, could you clarify, did you mean a PR to this repo or to cwl-utils?

Reply:
@tetron I meant a PR to this repo since I'm done with all the tasks mentioned in the README.

Reply:
Sorry, I was confused since we are already having this discussion on a pull request, and because I didn't see a program to read WDL from task (3) in this PR.  Please go ahead and update this one, or close it and create a pull request with both programs.  I will review it, and then we can talk about the next set of tasks to work on.

Reply:
ok I have created a new pull request having all the work completed in the tasks.